nav2_system_tests/src/localization/test_localization_node.cpp

```diff
@@ -81,15 +82,15 @@ class TestAmclPose : public ::testing::Test
 
 bool TestAmclPose::defaultAmclTest()
 {
-  initial_pose_pub_->publish(testPose_);
+  initial_pose_pub_->publish(*testPose_);
```

Comment:
same here `initial_pose_pub_->publish(std::move(*testPose_))`?

Reply:
Not quite, you're moving the pointer ownership, not the object.